samples/quickperf/src/main/java/com/google/cloud/jdbc/quickperf/QuickPerf.java

```diff
@@ -0,0 +1,497 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.jdbc.quickperf;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.security.SecureRandom;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.cli.*;
+import org.apache.commons.lang3.ArrayUtils;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.google.cloud.jdbc.quickperf.config.Config;
+import com.google.cloud.jdbc.quickperf.config.ConfigParser;
+import com.google.cloud.jdbc.quickperf.config.QueryParam;
+import com.google.cloud.spanner.Dialect;
+import com.google.cloud.spanner.jdbc.CloudSpannerJdbcConnection;
+
+import net.datafaker.Faker;
+
+public class QuickPerf extends Thread {
```

Comment:
I think that it would be good split this into two classes:
1. A `QuickPerfApplication` class that is responsible for running the application, reading the config etc.
2. A `QuickPerfRunner` class that extends `Thread`, and that does the actual performance benchmark.